src/inverseRobot.js

```diff
@@ -4,10 +4,20 @@
  * @param {object} robot
  *
  * @return {object}
-*/
+ */
 
 function inverseRobot(robot) {
   // write code here
+  const revese = {};
+
+  for (const keys in robot) {
+    if (revese.hasOwnProperty(robot[keys])) {
+      return null;
+    }
+    revese[robot[keys]] = keys;
```

Comment:
A blank line between a block and the next command

```suggestion
    }
    
    revese[robot[keys]] = keys;
```